Core/HLE/sceNet.cpp

```diff
@@ -450,7 +450,7 @@ void __NetApctlCallbacks()
 	// How AP works probably like this: Game use sceNetApctl function -> sceNetApctl let the hardware know and do their's thing and have a new State -> Let the game know the resulting State through Event on their handler
 	if (!apctlEvents.empty())
 	{
-		auto args = apctlEvents.front();
+		auto& args = apctlEvents.front();
```

Comment:
As i remembered this is an intentional copy since below it the element is being popped out (`apctlEvents.pop_front();`), i wasn't sure whether it's safe to reference a popped out element thus not using `auto&`.
Hopefully it's safe.